src/PKSim.Assets.Images/PKSim.Assets.Images.csproj

```diff
@@ -26,8 +26,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="OSPSuite.Assets" Version="11.1.65" />
-    <PackageReference Include="OSPSuite.Assets.Images" Version="11.1.65" />
+    <PackageReference Include="OSPSuite.Assets" Version="12.0.0-SisUj" />
```

Comment:
no please. Let's not create PR referencing unknow number

Reply:
@georgeDaskalakis You really have to check this out. I'd really appreciate it if you would review the PR (by going to the files) before asking me to review.

Reply:
@msevestre sorry for that, you are right. My suggestion here would be: since we are using this made up version of the nugets locally, should we actually add the .csproj to .gitignore (ONLY for this branch of course).
If we don't I can guarantee you this will happen again: like yesterday: if I review the changes and then need to adjust a small thing in a file, and not stage only that file we will end up getting all these changes in the project. I am not saying I will not try to be carefull, but it will definetely happen again.
@rwmcintosh what do you think about the gitignore?

Reply:
This is my fault. We can always back out the CSProj for PR's but the issue will be that when you clone the repo, the stuff won't build anyway because there is no centrally available ospsuite.core libraries that match.

This is a drawback of making your own local nuget repo.

For me, it is not a big difference if you leave this saying 11.1.65 or 12.0.0-fdjka because in both cases you need to create your own OSPSuite core before you can do anything.

Reply:
I wouldn't add them to .gitignore, because if you add a new file to a project, that change won't be available for the next person.
It's whack-a-mole. It makes debugging much easier, but introduced a new problem.

I'd sooner have the non-existent core number.

Although, at the moment, I'm the only one using this new method of producing core builds.

Reply:
the files paths are not stored in the .csproj. we went through the files with @rwmcintosh and actually the gitignore idea could work. @msevestre @Yuri05 let's discuss on monday

Reply:
I do not understand the reason why we would want to commit to our main `OSMOSES` branch code that does not compile. This does not make sense to me so I am probably missing the point. 
In my mind, osmoses is like develop. It should compile and be runnable at ALL time. So we cannot commit anything that is going to break. You can create as many branches as you want, if the goal is to share code etc, but in osmoses...I would expect this stuff to build.

Which means that we need an osmoses build of ospsuite.core that builds also where we do all of our PR. 
I'd love to understand the reason because there is probably some frictions going on that I do not see

Reply:
We would need the nuget repo to be centrally created and stored along with numbering etc. We turned that off for OSMOSES branch due to conflicts with the build numbering while we are working on develop at the same time.

If they repositories can be somehow independent that would be another way. But based on the fact that Core is changing, and there's no central NuGet repo the developer will always have to create a Core build that works before they can build the apps.

Until we merge OSMOSES to develop, we will always have either the friction of no central Core NuGet, or conflicting (or alternating) Core build numbers between develop and OSMOSES

Reply:
I understand what we need but I don't understand why we would want to merge non compiling code. 
What is the `purpose` of this?

>(or alternating) Core build numbers between develop and OSMOSES

This is fine with me. We will reset all the counters as soon as we release 11.1 anyways

Reply:
> What is the purpose of this?

As soon as we turned off the nuget build of OSPSuite.Core OSMOSES branch on AppVeyor this was going to happen. The purpose was to not create build number confusion with develop.

Reply:
You just need to run the batch file to create your local repo to get it working.

In any case, if you think the core builds should be turned back on then that's fine too.

The whole purpose was to avoid the build numbering conflict, but if the cost is too great, then fine.  But it's one or the other.

Reply:
I just don't get it. Let's discuss on Monday